src/sentry/utils/uwsgi.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
+
+import os
+
+
+def filemon(path):
```

Comment:
Without the uWSGI background context, a like `reload_on_file_change` or something would be clearer here, `filemon` is pretty abstract and doesn't really communicate the action being taken here